backend-rust/src/indexer.rs

```diff
@@ -2565,3 +2751,146 @@ impl PreparedScheduledReleases {
         Ok(())
     }
 }
+
+// Extends the `Account` type by including the `AccountIndex`.
+pub struct AccountExtended {
```

Comment:
Can we avoid having these types by fetching the index from the table during processing?
[Just like we do here](https://github.com/Concordium/concordium-scan/blob/9d6255d16e5b1ece5326030f9d6087ffbd28ab52/backend-rust/src/indexer.rs#L1035)